src/Database.m

```diff
@@ -134,7 +134,7 @@ - (BOOL)initialiseDatabase {
     } else if (databaseVersion >= MA_Min_Supported_DB_Version) {
         NSAlert * alert = [[NSAlert alloc] init];
         [alert setMessageText:NSLocalizedString(@"Database Upgrade", nil)];
-        [alert setInformativeText:NSLocalizedString(@"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconveninece.", nil)];
+        [alert setInformativeText:NSLocalizedString(@"Database upgrade explanation text", @"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconvenience.")];
```

Comment:
Did you read the objc.io article I pointed to ? The example with the possible meanings of the "run" idiom within a fitness application seems quite convincing to me.

Also, you have to take into account that, by its nature, the translation process must be a much more decentralized process than pure coding.
Judging from my experience with Crowdin's French suggestions, I would be very cautious and I am certain I will never risk going through a Turkish, Russian or Ukrainian file in order to decide about an expression… There are enough geopolitical risks in our world, a linguistic error could put me on a black list for getting a visa to visit some countries 🤣